config/routes.rb

```diff
@@ -1,9 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get '/api/users/:user_id/posts/', to: 'api_endpoints#user_posts'
+  get '/api/users/:user_id/posts/:post_id/comments', to: 'api_endpoints#user_post_comments'
 
```

Comment:
Kindly, add API endpoint that adds' a comment to a post which should be `POST` request. It will be the same as line 4 get request but the only change will be it will `POST` request. It will be something like this: 👇 

```
post '/api/users/:user_id/posts/:post_id/comments', ....
```

Reply:
Hi @Mre55 max contributed to the project. We worked in the morning together. You can see his comments at the start. Then we got stuck after 4 hours of work. So we decided to tackle the issue separately.

Thats why you see the branch is named test. Initially it was api endpoints.

As for the API endpoint.. The comment one is handles in the create action inside the comment controller.